lib/openxml/element.rb

```diff
@@ -10,17 +10,18 @@ class << self
 
       def tag(*args)
         @tag = args.first if args.any?
-        @tag
+        @tag ||= nil
       end
 
       def name(*args)
         @property_name = args.first if args.any?
-        @name
+        @property_name ||= nil
       end
 
+      alias _attribute_namespace namespace
```

Comment:
🛑 Heh heh. This one's unnecessary. It's complaining because there's an `attr_reader :namespace` on line 9! You can just remove the unnecessary `attr_reader`.

Reply:
I'll make that change and force-push